.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+default_language_version:
+  python: python3
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+    
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
```

Comment:
run `pre-commit autoupdate` on this repo (for more info check out doc of pre-commit